packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -1951,6 +1951,35 @@ describe("Teal Opcodes", function () {
         assert.equal(1, stack.length());
         assert.deepEqual(TXN_OBJ.apsu, stack.pop());
       });
+
+      it("should push value from accounts or args array by index", function () {
```

Comment:
please add test for `ApplicationArgs`.

Reply:
[https://github.com/scale-it/algorand-builder/pull/248/files/71ca98c1ebf4be194883bca6637fff28fc203b9e#diff-0bd11169d40e3be81fef88fe8a8e4cf0795255b19d1f4d44d615ea872a1e0e90R1977](https://github.com/scale-it/algorand-builder/pull/248/files/71ca98c1ebf4be194883bca6637fff28fc203b9e#diff-0bd11169d40e3be81fef88fe8a8e4cf0795255b19d1f4d44d615ea872a1e0e90R1977)